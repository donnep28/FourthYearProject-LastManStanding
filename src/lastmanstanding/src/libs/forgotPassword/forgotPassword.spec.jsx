import React from 'react';
import {shallow} from 'enzyme';
import ForgotPassword from './forgotPassword';

describe('forgotPassword', () => {
  beforeAll(() => {
    jest.resetAllMocks();
  });

  describe('Snapshot', () => {
    it('renders correctly', () => {
      const component = shallow(
          <ForgotPassword />,
      );
      expect(component).toMatchSnapshot();
    });
  });
});
