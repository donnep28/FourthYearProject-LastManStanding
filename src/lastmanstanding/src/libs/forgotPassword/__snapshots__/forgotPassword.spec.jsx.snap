// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`forgotPassword Snapshot renders correctly 1`] = `
<UsernameModal
  getBack={[Function]}
  handleCodeSent={[Function]}
  setUsername={[Function]}
  username=""
/>
`;
